frontend/src/components/InputField/InputField.module.css

```diff
@@ -32,7 +32,7 @@
 
 .input.medium {
   height: 56px;
-  width: 314px;
```

Comment:
Are you sure we need to change medium size? Afetr this change medium and large size are gonna be almost the same (4px difference)